microcosm_caching/memcached.py

```diff
@@ -65,31 +66,38 @@ class MemcachedCache(CacheBase):
     """
     def __init__(
         self,
-        host="localhost",
-        port=11211,
+        servers: Optional[Tuple[str, int]] = None,
         connect_timeout=None,
         read_timeout=None,
         serializer=json_serializer,
         deserializer=json_deserializer,
         testing=False,
     ):
         client_kwargs = dict(
-            server=(host, port),
             connect_timeout=connect_timeout,
             timeout=read_timeout,
-            serializer=json_serializer,
-            deserializer=json_deserializer,
+            serializer=serializer,
+            deserializer=deserializer,
         )
+
         if testing:
-            self.client = MockMemcacheClient(**client_kwargs)
+            self.client = MockMemcacheClient(
+                server=None,
+                **client_kwargs,
+            )
         else:
-            self.client = Client(**client_kwargs)
+            self.client = HashClient(
+                servers=servers,
+                **client_kwargs,
+            )
 
-    def get(self, key):
+    def get(self, key: str):
         return self.client.get(key)
 
-    def set(self, key, value, ttl=None):
+    def set(self, key: str, value, ttl=None):
         if ttl is None:
             # pymemcache interprets 0 as no expiration
             ttl = 0
+        # NB: If input is malformed, this will not raise errors.
+        # set `noreply` to False for further debugging
```

Comment:
I opted not to disable this wholesale for the moment; may be with making configurable?